### PR TITLE
add `PerlBundle` generic easyblock for installing a bundle of Perl modules

### PR DESCRIPTION
**Pull request by Micket**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyblocks/generic/perlbundle.py [hunk 1]
```diff
@@ -0,0 +1,94 @@
+##
+# Copyright 2018-2021 Ghent University
```
Comment (boegel) [inline easybuild/easyblocks/generic/perlbundle.py:2]:
```suggestion
# Copyright 2018-2023 Ghent University

**Review by boegel — CHANGES_REQUESTED** [review]

**Review by boegel — APPROVED** [review]

File: easybuild/easyblocks/generic/perlbundle.py [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+        # combine custom easyconfig parameters of Bundle & PerlModule
+        extra_vars = PerlModule.extra_options()
+        return Bundle.extra_options(extra_vars)
+
+    def __init__(self, *args, **kwargs):
+        """Initialize PerlBundle easyblock."""
+        super(PerlBundle, self).__init__(*args, **kwargs)
+
+        self.cfg['exts_defaultclass'] = 'PerlModule'
+        self.cfg['exts_filter'] = ("perl -e 'require %(ext_name)s'", '')
```
Comment (Micket) [inline easybuild/easyblocks/generic/perlbundle.py:56]: If the perl easyblock uses this as well we could import the variable. I thought it more important to do the right thing from the start here.

**Comment by boegel** [comment]
tested with https://github.com/easybuilders/easybuild-easyconfigs/pull/17970, ready to go, thanks @Micket!

**Comment by Flamefire** [comment]
This causes trouble on Python2 due to the umlaut --> https://github.com/easybuilders/easybuild-easyblocks/pull/3008

**Pull request merged**